pyperformance doesn't work on the current Python 3.11 dev version: fail to install greenlet

```
$ ./bin/python3.11 -m pyperformance run -o ~/python/pgo_lto_pyperformance_macros.json -v 
(...)
    creating build/temp.linux-x86_64-3.11/src/greenlet
    gcc -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/home/vstinner/python/main/install/venv/cpython3.11-527ffb5582f0/include -I/home/vstinner/python/main/install/include/python3.11 -c src/greenlet/greenlet.c -o build/temp.linux-x86_64-3.11/src/greenlet/greenlet.o
    src/greenlet/greenlet.c: In function ‘g_switchstack’:
    src/greenlet/greenlet.c:508:44: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘recursion_depth’; did you mean ‘recursion_limit’?
      508 |         current->recursion_depth = tstate->recursion_depth;
          |                                            ^~~~~~~~~~~~~~~
          |                                            recursion_limit
    src/greenlet/greenlet.c:509:38: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘frame’; did you mean ‘cframe’?
      509 |         current->top_frame = tstate->frame;
          |                                      ^~~~~
          |                                      cframe
    src/greenlet/greenlet.c:544:17: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘recursion_depth’; did you mean ‘recursion_limit’?
      544 |         tstate->recursion_depth = target->recursion_depth;
          |                 ^~~~~~~~~~~~~~~
          |                 recursion_limit
    src/greenlet/greenlet.c:545:17: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘frame’; did you mean ‘cframe’?
      545 |         tstate->frame = target->top_frame;
          |                 ^~~~~
          |                 cframe
    src/greenlet/greenlet.c: In function ‘g_initialstub’:
    src/greenlet/greenlet.c:821:50: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘recursion_depth’; did you mean ‘recursion_limit’?
      821 |     self->recursion_depth = PyThreadState_GET()->recursion_depth;
          |                                                  ^~~~~~~~~~~~~~~
          |                                                  recursion_limit
    error: command '/usr/bin/gcc' failed with exit code 1
```

--- a/pyperformance/data-files/benchmarks/bm_genshi/requirements.txt
+++ b/pyperformance/data-files/benchmarks/bm_genshi/requirements.txt
@@ -1,3 +1,2 @@
-# genshi 0.7.5+:
-git+git://github.com/edgewall/genshi.git@d5975012c4175bcca16ccf4928fdd54dcd211940
+genshi==0.7.6
 six==1.16.0

--- a/pyperformance/data-files/benchmarks/bm_genshi/requirements.txt
+++ b/pyperformance/data-files/benchmarks/bm_genshi/requirements.txt
@@ -1,2 +1,3 @@
-genshi==0.7.5
+# genshi 0.7.5+:
+git+git://github.com/edgewall/genshi.git@d5975012c4175bcca16ccf4928fdd54dcd211940
 six==1.16.0

--- a/pyperformance/data-files/benchmarks/bm_sqlalchemy_declarative/requirements.txt
+++ b/pyperformance/data-files/benchmarks/bm_sqlalchemy_declarative/requirements.txt
@@ -1,2 +1,2 @@
-greenlet==1.1.0
+greenlet==2.0.0a1
 sqlalchemy==1.4.19

--- a/pyperformance/data-files/benchmarks/bm_sqlalchemy_imperative/requirements.txt
+++ b/pyperformance/data-files/benchmarks/bm_sqlalchemy_imperative/requirements.txt
@@ -1,2 +1,2 @@
-greenlet==1.1.0
+greenlet==2.0.0a1
 sqlalchemy==1.4.19

--- a/setup.py
+++ b/setup.py
@@ -3,7 +3,6 @@
 # FIXME:
 #
 # - REENABLE HG_STARTUP BENCHMARK.
-# - REENABLE GENSHI BENCHMARK.
 #
 # Update dependencies:
 #

